fabric/v4/model_network.go

```diff
@@ -19,4 +19,17 @@ type Network struct {
 	Project  *Project            `json:"project,omitempty"`
 	// Preferences for notifications on network configuration or status changes
 	Notifications []SimplifiedNotification `json:"notifications"`
+	// Network URI
+	Href string `json:"href"`
```

Comment:
For changes like this, we've leaned towards spec patches (using the `make patch` task) rather than post patches; that makes it easier to get the changes adopted upstream, since we've already figured out what the spec needs to look like in order to achieve the desired generator output.  It also makes these sorts of changes easier to propagate across SDKs (in the event that fabric-java needs this stuff too).

Reply:
This is a smarter move. I'll update that.